modules/event-hub/namespaces/main.bicep

```diff
@@ -9,10 +9,11 @@ param location string = resourceGroup().location
 @allowed([
   'Basic'
   'Standard'
+  'Premium'
 ])
 param skuName string = 'Standard'
 
-@description('Optional. Event Hub plan scale-out capacity of the resource.')
+@description('Optional. The Event Hubs throughput units for Basic or Standard tiers, where value should be 0 to 20 throughput units. The Event Hubs premium units for Premium tier, where value should be 0 to 10 premium units.')
```

Comment:
```suggestion
@description('Optional. The Event Hub''s throughput units for Basic or Standard tiers, where value should be 0 to 20 throughput units. The Event Hubs premium units for Premium tier, where value should be 0 to 10 premium units.')
```

Reply:
Just one small suggestion. Please don't forget to update the readme too.

Reply:
@AlexanderSehr done :) Ready for review